creation/lib/cWDictFile.py

```diff
@@ -1411,6 +1409,7 @@ def add_extended(
         elif user_name == True:
             user_name = "+"
 
+        # TODO: check .add end set allow_overwrite=False above instead allow_overwrite=0
```

Comment:
I believe there's a typo here: "end" instead of "and". Definitely not a show stopper.
```suggestion
        # TODO: check .add and set allow_overwrite=False above instead allow_overwrite=0
```

Reply:
done